.cirrus.yml

```diff
@@ -71,7 +71,7 @@ task:
           CHANNEL: "2.5.3"
           CHANNEL: "2.8.1"
       analyze_script:
-        - ./script/tool_runner.sh analyze --skip-if-not-supporting-flutter-version="$CHANNEL" --custom-analysis=script/configs/custom_analysis.yaml
+        - ./script/tool_runner.sh analyze --skip-if-not-supporting-flutter-version="$CHANNEL" --custom-analysis=script/configs/custom_analysis.yaml --exclude=flutter_lints
```

Comment:
Please put a comment explaining why it's excluded (see elsewhere in the file that uses `--exclude`), so that the CI script is self-documenting about why we have exceptions.